docker-compose.yml

```diff
@@ -0,0 +1,107 @@
+---
+version: '3.7'
+services:
+
+  imsearch:
+    image: imsearch-cpu:latest
+    container_name: ${NAMESPACE}-cpu
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+    - 5000:5000
+    environment:
+    - MONGO_URI=mongodb://mongodb:27017/
+    - REDIS_URI=redis://redis:6379/0
+    - STORAGE_MODE=local
+    depends_on:
+    - redis
+    - mongodb  
+    links:
+    - mongodb:mongodb
+    - redis:redis
+    networks:
+    - internal
+    - web
+
+  mongodb:
+    image: webhippie/mongodb:latest
```

Comment:
Why are we using this image? Can't we just use official [`mongo:latest`](https://hub.docker.com/_/mongo) image?

Reply:
I used it because it is alpine based and slimmer.
Ref: https://hub.docker.com/r/webhippie/mongodb/

Reply:
Sure. we can use this one.